cime_config/tests.py

```diff
@@ -162,6 +162,7 @@
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld1.ne30_r05_oECv3.A_WCYCL1850",
             "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-genmmf",
+            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_variance_transport",
```

Comment:
Maybe we should just replace the other MMFXX test with this? 
Having two doesn't really increase our coverage here.

Reply:
Removed.